packages/tradershub/src/features/cfd/screens/VerificationFailed/VerificationFailed.tsx

```diff
@@ -1,52 +1,38 @@
 import React from 'react';
-import { usePOA, usePOI } from '@deriv/api';
 import { Provider } from '@deriv/library';
 import { Button, Text } from '@deriv/quill-design';
 
-const getDocumentTitle = (isPOIFailed?: boolean, isPOAFailed?: boolean) => {
-    if (isPOIFailed && isPOAFailed) return 'proof of identity and proof of address';
-    if (isPOIFailed) return 'proof of identity';
-    return 'proof of address';
-};
-
-const reasons = ['Document details do not match profile details', 'Expired documents', 'Poor image quality'];
+const reasons = ['Proof of identity', 'Proof of address'];
```

Comment:
why did you remove these?

Reply:
@mahdiyeh-deriv I followed the design attached in the channel https://www.figma.com/file/p3d3hK27vuQtA4BDhTjjpp/Release-(Rebuild)---Low-risk-ROW-Trader%E2%80%99s-hub?type=design&node-id=889-76296&mode=dev

Reply:
@sergei-deriv the content should be dynamic based on the POI, POA statuses so it still needs to check for statuses to get the title and content

Reply:
@mahdiyeh-deriv Ok, in this case if it's out of the scope of this card, I'll just keep the `VerificationFailed` component unchanged and create just `VerificationFailedModal`, is it ok?

Reply:
@mahdiyeh-deriv I'll leave `VerificationFailed` component unchanged